src/ffi/mod.rs

```diff
@@ -4,6 +4,10 @@
 //! It is meant for advanced users only - regular PyO3 users shouldn't
 //! need to interact with this module at all.
 //!
+//! The contents of this module are not documented here, as it would entail
+//! keeping the documentation up to date. Consult the [Python/C API Reference
```

Comment:
I know what you mean, but it reads a bit funny - of course documentation always has to be kept up to date :)

```suggestion
//! basically copying the documentation from CPython. Consult the [Python/C API Reference
```